ui-speedspacechart/src/stories/SpeedSpaceChart.stories.ts

```diff
@@ -0,0 +1,29 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import '@osrd-project/ui-core/dist/theme.css';
+import '@osrd-project/ui-speedspacechart/dist/theme.css';
+import SpeedSpaceChart from '../components/SpeedSpaceChart';
+import OSRD_SAMPLE from './assets/sampleData';
+
+const meta: Meta<typeof SpeedSpaceChart> = {
+  title: 'SpeedSpaceChart',
```

Comment:
```suggestion
  title: 'SpeedSpaceChart/Rendering',
```
To have its own section in the stories menu

Reply:
of course, done